mathics/builtin/trace.py

```diff
@@ -354,11 +355,13 @@ class TraceEvaluation(Builtin):
       <dd>Evaluate $expr$ and print each step of the evaluation.
     </dl>
 
-    >> TraceEvaluation[(x + x)^2]
+    The 'ShowTimeBySteps' option prints the elapsed time before an evaluation occurs.
+
+    >> TraceEvaluation[(x + 1)^2]
```

Comment:
Note this gives less verbose trace output since we don't have to look up x again.

Reply:
@mmatera would like to merge this in. The next step will be something similar where we decorate the rewrite-apply step so we can see what's going on there. It will be similar to this.